Cargo.toml

```diff
@@ -20,7 +20,7 @@ libc = "0.2.78"
 winapi = { version = "0.3.8", features = ["std", "handleapi", "namedpipeapi", "processenv", "synchapi", "winerror", "processthreadsapi", "winbase"] }
 
 [dev-dependencies]
-tempdir = "0.3.7"
+tempfile = "3"
```

Comment:
Let's rather specify `"3.3.0"` (3.3.0 or newer), for reasons [explained here](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277).

Reply:
thanks for the feedback :)